### PR TITLE
Release/0.5

### PR DESCRIPTION
**Pull request by muralikarthikk**

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -14,9 +14,14 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.4.0
+version: 0.5.rc0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 1.2.0
+appVersion: 1.3.dev-c6f479bc
```
Comment (cschreep) [inline Chart.yaml:21]: I think, for merges into the main branch, we should get the appstore repo a proper release, is there anything preventing us from that?

**Review by cschreep — CHANGES_REQUESTED** [review]

**Comment by pj-linebaugh** [comment]
Maybe do a release after these latest PRs?  I have a small PR to submit too, which I will right after this.

**Comment by cschreep** [comment]
> Maybe do a release after these latest PRs? I have a small PR to submit too, which I will right after this.

Sounds good to me

**Review by pj-linebaugh — APPROVED** [review]

**Comment by pj-linebaugh** [comment]
@muralikarthikk  Images can be built from the appstore master branch now.  Can this move forward?  or maybe just restart it?

**Pull request closed without merging**